src/agent/create-test-runner.js

```diff
@@ -36,15 +36,15 @@ export async function createRunner(options) {
       browser: options.webDriverBrowser,
       abortSignal: options.abortSignal,
       log: options.log,
-    }).catch(err => {
-      throw new Error(`Error connecting to web-driver: ${err}`);
+    }).catch(cause => {
+      throw new Error('Error connecting to web-driver', { cause });
     }),
     createATDriver({
       url: options.atDriverUrl,
       abortSignal: options.abortSignal,
       log: options.log,
-    }).catch(err => {
-      throw new Error(`Error connecting to at-driver: ${err}`);
+    }).catch(cause => {
+      throw new Error('Rrror connecting to at-driver', { cause });
```

Comment:
![image](https://github.com/w3c/aria-at-automation-harness/assets/677252/8ba01663-f0fe-4a51-96df-71b3c8671b15)

Reply:
yeah lul - forgot to hit save situation, fixed it locally but never added it to the commit Return of the Rrrors